### DISCUSSION TITLE
Your Copilot experience is not fully configured... loop

### DISCUSSION THREAD

**Discussion by georgalis** [community/community#19579] [Copilot Conversations] [answered]
Hi, I'm trying copilot in a sandbox (vm with new user) on centos 7 with nvim. The getting started instructions for nvim are rather vague "To install the GitHub Copilot plugin directly, you must know where Neovim stores plugins." https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-neovim#prerequisites

 I'm familiar with vim plugins, but nvim is rather different. Here is a better example for the doc:
```
mkdir -p .config/nvim/pack/github/start
git clone https://github.com/github/copilot.vim.git \
       ~/.config/nvim/pack/github/start/copilot.vim
```

I've installed nodejs v16.15.0 with yum from the epel-release repo.
```
yum install epel-release && yum install nodejs
```

The nvim from epel is not new enough, so I got the current stable release from 
https://github.com/neovim/neovim/releases/tag/stable and extracted it.
```
wget -N https://github.com/neovim/neovim/releases/download/stable/nvim-linux64.tar.gz
tar xzf nvim-linux64.tar.gz
```

For this setup round, I'll remove my prior auth token
```
rm ./.config/github-copilot/hosts.json
```

I'll edit a new file with my new editor and the copilot release branch I cloned above,
```
./nvim-linux64/bin/nvim new.cpp
```

nvim works just like vim, but I need to setup copilot, within nvim I enter <code>:Copilot setup</code> then copy the one-time code, complete the browser auth of the copilot plugin at https://github.com/login/device 
```
First copy your one-time code: XXXX-XXXX
Press ENTER to open GitHub in your browser
Opened https://github.com/login/device
Waiting (could take up to 5 seconds)
```

then, I'm presented with the following:
```
Your Copilot experience is not fully configured, complete your setup.
Your Copilot experience is not fully configured, complete your setup.

Request Actions:
1. Copilot Settings
2. Dismiss
Type number and <Enter> or click with the mouse (q or empty cancels):
```

upon entering 1, I'm presented with
```
Copilot: Authenticated as GitHub user georgalis
Press ENTER or type command to continue
:Copilot enable
```
Then every time I would expect to see copilot prompting, I get the above notice again "Your Copilot experience..." (twice) with a prompt for settings or dismiss... when I select settings, the prompt disappears and I return to edit mode.  When I manually invoke <code>:Copilot setup</code> I get the friendly response:

```
Copilot: Authenticated as GitHub user georgalis
Press ENTER or type command to continue
```

The setup prompt makes it impossible to edit while <code>:Copilot enabled</code>... perhaps the copilot clone needs to go in a different directory? If so, could we update the getting started with nvim doc to reflect using the nvim built in plugin manager?

**Comment by davecheney** [comment, marked as answer]
@georgalis go to https://github.com/settings/copilot and complete the required steps.

**Reply by georgalis** [reply]
Thanks @davecheney! 

I don't think that's mentioned in the setup docs? I created a pull request for it..
https://github.com/github/docs/pull/18833
